pkg/controller/controllerutil/controllerutil.go

```diff
@@ -95,7 +95,7 @@ func SetControllerReference(owner, object metav1.Object, scheme *runtime.Scheme)
 	}
 
 	// Update owner references
-	object.SetOwnerReferences(existingRefs)
+	owner.SetOwnerReferences(existingRefs)
```

Comment:
typo here.  I think this is a good reason that we should maybe stick with `owner` and `object`, or come up with something different -- maybe parent and child?  Otherwise it's too easy to typo

Reply:
:+1: would also like to see a test catching this.

Reply:
It turns out that the first test ("should set the OwnerReference if it can find the group version kind") catches this; I think this was a stray unlucky keystroke in overwrite mode which happened to be valid.

I've renamed `owned` to `controlled`, which should avoid that problem.